### PR TITLE
Update features edit sample

### PR DESCRIPTION
**Pull request by nanaeaubry**
Use Folder add instead of ContentManager add

### CODE REVIEW

File: samples/05_content_publishers/updating_features_in_a_feature_layer.ipynb [hunk 1]
```diff
@@ -1 +1 @@
... and you may choose this when the number of features to edit is less or if you needed to selectively update certain features as updates come in.\n", "\n", "An alternate method is to overwrite the feature layer altogether when you always have current information coming in. This method is explained in the sample [Overwriting feature layers](https://developers.arcgis.com/python/sample-notebooks/overwriting-feature-layers)"]}], "metadata": {"anaconda-cloud": {}, "esriNotebookRuntime": {"notebookRuntimeName": "ArcGIS Notebook Python 3 Standard", "notebookRuntimeVersion": "4.0"}, "kernelspec": {"display_name": "Python 3", "language": "python", "name": "python3"}, "language_info": {"codemirror_mode": {"name": "ipython", "version": 3}, "file_extension": ".py", "mimetype": "text/x-python", "name": "python", "nbconvert_exporter": "python", "pygments_lexer": "ipython3", "version": "3.8.2"}, "toc": {"base_numbering": 1, "nav_menu": {}, "number_sections": true, "sideBar": true, "skip_h1_title": false, "title_cell": "Table of Contents", "title_sidebar": "Contents", "toc_cell": true, "toc_position": {}, "toc_section_display": true, "toc_window_display": true}}, "nbformat": 4, "nbformat_minor": 1}
```
Comment (cariashuang0417) [inline samples/05_content_publishers/updating_features_in_a_feature_layer.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #2.    item_prop = {'title':'USA Capitals spreadsheet ' + now_ts}
</pre><p>Need <code>"item_type": "CSV"</code> for item properties as a required key value pair :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2137/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (cariashuang0417) [reply samples/05_content_publishers/updating_features_in_a_feature_layer.ipynb:1]: <p>Oops and <code>.result()</code> to get the Folder.add() result!</p>

**Review by cariashuang0417 — APPROVED** [review]

**Review by jyaistMap — CHANGES_REQUESTED** [review]
@nanaeaubry  - Can you please address the conflicts?

**Review by jyaistMap — APPROVED** [review]
looks good, thanks @nanaeaubry

**Pull request merged**